mon-pix/app/templates/user-tutorials.hbs

```diff
@@ -3,41 +3,45 @@
 <BurgerMenu @animation="push" @translucentOverlay={{true}} as |burger|>
   <burger.outlet>
     <NavbarHeader @burger={{burger}} />
-    <header role="banner" class="background-banner">
-      <div class="user-tutorials-banner">
-        <img src="{{rootURL}}/images/background/tutorials-banner.svg" class="user-tutorials-banner__icon" role="none" alt=""/>
-        <h3 class="user-tutorials-banner__title">{{t 'pages.user-tutorials.title'}}</h3>
-        <p class="user-tutorials-banner__description">
-          {{t 'pages.user-tutorials.description'}}
-        </p>
-      </div>
-    </header>
-    {{#if this.model}}
-      <article aria-label={{t 'pages.user-tutorials.label'}} class="user-tutorials-content">
-        <h3 class="user-tutorials-content__title">{{t 'pages.user-tutorials.list.title'}}</h3>
-        <div>
-          {{#each this.model as |userTutorial|}}
-            <TutorialItem @tutorial={{userTutorial.tutorial}} />
-          {{/each}}
-        </div>
-      </article>
-    {{else}}
-      <article aria-label={{t 'pages.user-tutorials.label'}} class="rounded-panel user-tutorials-no-tutorial">
-        <div class="user-tutorials-no-tutorial__illustration">
-          <img src="{{rootURL}}/{{t 'pages.user-tutorials.empty-list-info.image-link'}}"
-               role="none"
+    <main id={{t 'common.skip-links.link-to-content'}}>
+      <header role="banner" class="background-banner">
+        <div class="user-tutorials-banner">
+          <img src="{{rootURL}}/images/background/tutorials-banner.svg" class="user-tutorials-banner__icon" role="none"
```

Comment:
**question** Lui retirer la `alt=""` n'a pas d'impact ? j'ai vu plus bas dans le même fichier qu'il il y a une `img` en role none avec un alt vide. (L29.)

Reply:
Oui il manque un alt ici 👍 

D'ailleurs je pense que ajouter un `role` à une image qui a un alt vide est overkill.
Dans la doc W3, ils n'utilisent pas le alt vide et le role presentation ensemble sur une <img />. Ils utilisent l'un ou l'autre.

> Screen readers also allow the use of WAI-ARIA to hide elements by using role="presentation". However, currently, this feature is not as widely supported as using a null alt attribute.
https://www.w3.org/WAI/tutorials/images/decorative/

Reply:
Je n'ai pas retiré le `alt` : c'est le diff qui est chelou !👇 ligne 10 ^^

Reply:
@celineung le linter n'est pas content sans role :

> app/templates/user-tutorials.hbs
  9:10  error  If the `alt` attribute is present and the value is an empty string, `role="presentation"` or `role="none"` must be present  require-valid-alt-text
  28:12  error  If the `alt` attribute is present and the value is an empty string, `role="presentation"` or `role="none"` must be present  require-valid-alt-text